spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageResource.java

```diff
@@ -146,4 +136,14 @@ public String getDescription() {
 	public InputStream getInputStream() throws IOException {
 		return Channels.newInputStream(resolve().reader());
 	}
+
+	@Override
+	public boolean isWritable() {
+		return true;
+	}
+
+	@Override
+	public OutputStream getOutputStream() throws IOException {
+		return new GoogleStorageOutputStream(resolve());
```

Comment:
I wonder why doesn't `return Channels.newOutputStream(resolve().writer());` fit here?
Why do we need this custom `OutputStream` impl?

Thanks